src/data/finset/lattice.lean

```diff
@@ -1575,4 +1575,17 @@ lemma set_bInter_bUnion (s : finset γ) (t : γ → finset α) (f : α → set 
   (⋂ y ∈ s.bUnion t, f y) = ⋂ (x ∈ s) (y ∈ t x), f y :=
 infi_bUnion s t f
 
+lemma is_lub.finset_sup' {ι α : Sort*} [semilattice_sup α] {f : ι → α} {a : α}
```

Comment:
Is there an ambient `ι` or `α` that can be reused? I suspect that one or both of these is actually being adjusted to `Type` during elaboration.

Reply:
Yes, thanks!